### PR TITLE
Resolves #3

### LINKED ISSUE #3: Taylor Coury Test Issue
Git/GitHub for Sprint 0

### PR DESCRIPTION
**Pull request by tacoury**
Resolves #3

### CODE REVIEW

File: FlippedTech/homePage.js [hunk 1]
```diff
@@ -18,17 +18,17 @@ for(var i = 0; i < buttons.length; i++)
```
Comment (nakasoori) [inline FlippedTech/homePage.js]: Looks good

Comment (nakasoori) [inline FlippedTech/homePage.js:19]: Looks good

**Review by nakasoori — CHANGES_REQUESTED** [review]
Before we merge, reply to this comment

File: FlippedTech/homePage.js (hunk 1 above)
Comment (tacoury) [reply FlippedTech/homePage.js:19]: Thanks!

**Review by nakasoori — APPROVED** [review]

**Pull request merged**